File: EnforcerServices/src/main/java/com/webforged/enforcer/management/data/Project.java
package com.webforged.enforcer.management.data;

import java.time.LocalDateTime;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
@Table("projects")
public class Project {
	
	@Id
	public Long project_id ;
	private String acronym;
	private String business_owner;
	private String it_owner;
	private LocalDateTime begin_date;
	private LocalDateTime end_date;
	
	public Project() {
	}
}

File: EnforcerWeb/src/app/model/Project.ts
export class Project {
	public project_id: number;
	public acronym: string;
	public business_owner?: string;
	public it_owner?: string;
	public begin_date?: Date;
	public end_date?: Date;

	constructor( obj: Object = {}) {
		Object.assign(this, obj) ;
	}
 
	public toJSON() {
		return {
			project_id: this.project_id,
			acronym: this.acronym,
			business_owner: this.business_owner,
			it_owner: this.it_owner,
			begin_date: this.begin_date,
			end_date: this.end_date
		}
	}
}